tfx/benchmarks/tfma_v2_benchmark_base.py

```diff
@@ -135,8 +136,8 @@ def benchmarkMiniPipelineBatched(self):
     pipeline = self._create_beam_pipeline()
     tfx_io = test_util.InMemoryTFExampleRecord(
         schema=benchmark_utils.read_schema(
-            self._dataset.tf_metadata_schema_path()),
-        raw_record_column_name=constants.ARROW_INPUT_COLUMN)
+            "../examples/chicago_taxi_pipeline/data/user_provided_schema/schema.pbtxt"),
+        raw_record_column_name='__raw_record__')
```

Comment:
What do these changes do?

Reply:
For some reason, the dataset path stored in a cloud bucket for self._dataset.tf_metadata_schema_path() passed an path that was unreachable in the bucket, even though that exact path existed. I added the constant back in, I realized I had deleted an import that caused issues

Reply:
For some reason, the dataset path stored in a cloud bucket for self._dataset.tf_metadata_schema_path() passed an path that was unreachable in the bucket, even though that exact path existed. I added the constant back in, I realized I had deleted an import that caused issues